data/en/services.yml

```diff
@@ -3,8 +3,7 @@ services:
   enable : true
   title : Consulting Services
   subtitle: |
-    Building secure, performant, and scalable applications and data products is hard. We
-    want to make it easier and less costly.
+    "Rotational Labs has been a true partner to us -- from helping us realize and deploy our ambitious visions for next-gen supply chain analytics, to empowering our technical team with the skills and ability to maintain our products and services long-term." - Sebastian Sobolev, BlueVoyant Group
```

Comment:
I wasn't quite sure how to format the quote to make it look nice, but wanted to at least get the testimonial up. Maybe @daniellemaxwell can help me by adding a followon story for the future to do a bit nicer styling to the quote?

Reply:
Sure!